esmvalcore/_config/extra_facets/ipslcm-mappings.yml

```diff
@@ -0,0 +1,239 @@
+# Mapping, for ISPLCM output formats 'Analyse' and 'Output', between a
+# CMOR variable name and the labels to use by ESMValTool to find the
+# corresponding file, and the corresponding variable in the file
+#
+# For format 'Analyse', the config-development.yml file tells
+# ESMValTool to use key 'ipsl_varname' for building the filename,
+# while for format 'Output' it specifies to use key 'group'
+#
+# Specifying 'igcm_dir' here allows to avoid having to secifiy it in
+# datasets definitions
+#
+# Key 'use_cdo' allows to choose whether CDO will be invoked for
+# selecting a variable in a multi-variable file. This generally allows
+# for smaller overal load time. But because CDO has a licence which is
+# not compliant with ESMValtool licence policy, the default
+# configuration is to avoid using it. You may use customized settings
+# by installing a modified version of this file as
+# ~/.esmvatlool/variable_details/native6-ipsl-cm6-*.yml
+#
+# Key 'positive' allows to tell ESMValTool when the sign convention
+# for a variable is reversed between IPSL-CM6 and CMIP6. In that case,
+# just e.g. put 'positive: down' if the CMIP6 convention is 'positive: up'
+#
+# The main key below, 'IPSL-CM6', is the one to use as a value for
+# attribute 'dataset' in the 'datasets' entry of recipes; it matches
+# the module name 'ipsl.py' in 'cmor_fixes/ipsl*/ipsl.py'
```

Comment:
This is a bit confusing. Cleary `IPSL-CM6` *doesn't* match `ipsl.py`, so what's the deal?

Reply:
The comment was fine in an earlier version, where I defined two projects.
Fixed. Thanks. The issue with providing hints at multiple place is that redudancy leads to inconsistency...